### DISCUSSION TITLE
Are GitHub classic projects gone?

### DISCUSSION THREAD

**Discussion by harrydowning** [community/community#30227] [Projects and Issues] [answered]
I've added classic projects in the past but now, on new repositories, I don't seem to be able to. I would use the new projects but I don't like how the UI is detached from a repository.

My questions are:
- Is there some way to create classic projects or are they gone completely?
- Can the new projects be directly viewed from within a repository or does it always have to be viewed on a separate page?

**Comment by ichenglin** [comment]


**Reply by harrydowning** [reply]
I'm getting a 404 from your link and I only have the option for 'Projects', I can't see 'Projects (classic)'

**Reply by ministic2001** [reply]
I second this. I am also trying to find project classic board but found that it no longer exist even though I find it better than the current project board

**Comment by hermandebeukelaer** [comment, marked as answer]
We also bumped into this today. It appears from the [docs](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards) that classic projects are indeed being discontinued. You can "only create a new classic project board for an organisation, user or repository that already has at least one classic project board". Pretty annoying! Especially because the new projects do not exist at repo level...

**Reply by Splines** [reply]
See also the [sunset notice](https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/). Weird that they still have the checkbox in the settings page though.

**Comment by Katsute** [comment]
It's unclear if they are even adding repo level projects #6229.

As a workaround open an existing classic project that you own and copy it to the new repository.

**Reply by hermandebeukelaer** [reply]
Thanks for the suggested workaround, I will try it out today. I didn't know you could copy projects across repos.

**Comment by itma** [comment]
Hey,

Same problem here. I try to create a project via API getting the message `"{"message":"Projects are disabled for this organization","documentation_url":"https://docs.github.com/v3/projects"}"`

I have projects enabled for my org, yet cannot find where to create a classic project so that I can use API.

**Reply by Katsute** [reply]
They are blocking this feature even on the API level.

[![image](https://user-images.githubusercontent.com/58778985/188197215-f7259653-3450-44e4-afef-4c9965d36079.png)](https://docs.github.com/en/rest/projects/projects#create-a-repository-project)

https://docs.github.com/en/rest/projects/projects#create-a-repository-project

**Reply by itma** [reply]
Honestly I don't know what to do. Hoped I could use those Projects for managing multiple org projects from CRM yet it looks like not possible for now.

**Comment by xihajun** [comment]
also cannot find the old projects?
